extension-manifest-v3/package.json

```diff
@@ -37,8 +38,8 @@
     "@cliqz/adblocker": "^1.26.2",
     "@cliqz/adblocker-webextension-cosmetics": "^1.22.7",
     "@duckduckgo/autoconsent": "^4.0.0",
-    "@ghostery/libs": "file:../libs",
-    "@ghostery/ui": "file:../ui",
+    "@ghostery/libs": "^1.0.0",
```

Comment:
does this means that we will have to update those manually?

Reply:
Nope, this is one of the reasons behind the workspaces - you set it as it would go from npm, but it is linked in node_modules instead. The `^` satisfy everything until a major bump, so everything will work as expected.